src/main/java/nl/dtls/fairdatapoint/api/controller/MetadataController.java

```diff
@@ -88,83 +86,75 @@
 @RequestMapping("${urlPath.root:/fdp}")
 public class MetadataController {
 
-    private final static Logger LOGGER
-            = LogManager.getLogger(MetadataController.class);
+    private final static Logger LOGGER = LogManager.getLogger(MetadataController.class);
```

Comment:
Which logging framework are you using, and why is it not slf4j?

Reply:
@kburger  in general we are using log4j not?

Reply:
Maybe personal preference, but I'd propose slf4j as our standard. That way we can swap out implementation depending on our need (slf4j-simple for during tests, high performance log4j2 async during production).

Reply:
Yes, we are using log4j in the current implementation. I don't have any arguments not to use slf4j but shall we use it later? We might end up spending time if we swap it now.

Reply:
Yeah that's fine, I'll put it on the backlog. Ironically, when using slf4j you spent zero time swapping out implementations 😀.